### ISSUE TITLE
Error: Failure while executing; `/bin/launchctl bootstrap gui/501 /Users/yangzhou/Library/LaunchAgents/homebrew.mxcl.spacebar.plist` exited with 5.

### ISSUE THREAD

**Issue by fightyz** [Homebrew/brew#13728]
### `brew config` output

```shell
$ brew config
HOMEBREW_VERSION: 3.5.9
ORIGIN: https://github.com/Homebrew/brew
HEAD: 3748bed378401ed75abdf32bcb3d2674d854a6f9
Last commit: 11 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 96a45c178c0a18d382a3e5e68dc0008cd91fea7c
Core tap last commit: 13 hours ago
Core tap branch: master
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 2.6.8 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 13.1.6 build 1316
Git: 2.32.1 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.79.1 => /usr/bin/curl
macOS: 12.5-arm64
CLT: 13.4.0.0.1.1651278267
Xcode: 13.4.1
Rosetta 2: false
```

### `brew doctor` output

```shell
$ brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: "config" scripts exist outside your system or Homebrew directories.
`./configure` scripts often look for *-config scripts to determine if
software packages are installed, and which additional flags to use when
compiling and linking.

Having additional scripts in your path can confuse software installed via
Homebrew if the config script overrides a system or Homebrew-provided
script of the same name. We found the following "config" scripts:
  /opt/homebrew/Caskroom/miniforge/base/bin/python3.9-config
  /opt/homebrew/Caskroom/miniforge/base/bin/python3-config
  /opt/homebrew/Caskroom/miniforge/base/bin/ncursesw6-config
  /opt/homebrew/Caskroom/miniforge/base/bin/ncurses6-config

Warning: You have unlinked kegs in your Cellar.
Leaving kegs unlinked can lead to build-trouble and cause formulae that depend on
those kegs to fail to run properly once built. Run `brew link` on these:
  python@3.10
```

### Verification

- [X] I ran `brew update` and am still able to reproduce my issue.
- [X] I have resolved all warnings from `brew doctor` and that did not fix my problem.

### What were you trying to do (and why)?

Tried running `brew services start spacebar` and get an error with the plist file.

Similar to [issues-13678](https://github.com/Homebrew/brew/issues/13678), but this time is `gui/501`.

### What happened (include all command output)?

$ brew install cmacrae/formulae/spacebar
Running `brew update --auto-update`...
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/cask).

You have 2 outdated formulae and 1 outdated cask installed.
You can upgrade them with brew upgrade
or list them with brew outdated.

==> Downloading https://github.com/cmacrae/spacebar/releases/download/v1.4
Already downloaded: /Users/yangzhou/Library/Caches/Homebrew/downloads/dad899f372138dcbc86dffda80ad03f1ed90e5a3b7d165a59754caa2c76635a1--spacebar-v1.4.0.tar.gz
==> Installing spacebar from cmacrae/formulae
==> Caveats
Copy the example configuration into your home directory:
  cp /opt/homebrew/opt/spacebar/share/spacebar/examples/spacebarrc ~/.spacebarrc
Logs will be found in
  /opt/homebrew/var/log/spacebar/spacebar.[out|err].log

To restart cmacrae/formulae/spacebar after an upgrade:
  brew services restart cmacrae/formulae/spacebar
Or, if you don't want/need a background service you can just run:
  spacebar
==> Summary
🍺  /opt/homebrew/Cellar/spacebar/1.4.0: 6 files, 156.0KB, built in 1 second
==> Running `brew cleanup spacebar`...
Disable this behaviour by setting HOMEBREW_NO_INSTALL_CLEANUP.
Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).

$ brew services start spacebar
Bootstrap failed: 5: Input/output error
Try re-running the command as root for richer errors.
Error: Failure while executing; `/bin/launchctl bootstrap gui/501 /Users/yangzhou/Library/LaunchAgents/homebrew.mxcl.spacebar.plist` exited with 5.

$ brew services
Name     Status       User     File
skhd     started      yangzhou ~/Library/LaunchAgents/homebrew.mxcl.skhd.plist
spacebar error  19968 yangzhou ~/Library/LaunchAgents/homebrew.mxcl.spacebar.plist
yabai    started      yangzhou ~/Library/LaunchAgents/homebrew.mxcl.yabai.plist

### What did you expect to happen?

Start spacebar service successfully.

### Step-by-step reproduction instructions (by running `brew` commands)

```shell
brew install cmacrae/formulae/spacebar 
brew services start spacebar
```

**Comment by carlocab** [comment]
What's the output of
```sh
git -C "$(brew --repository homebrew/services)" rev-parse HEAD
```
?

**Comment by fightyz** [comment]
> What's the output of
> 
> ```shell
> git -C "$(brew --repository homebrew/services)" rev-parse HEAD
> ```
> 
> ?

a8f4e6d6d30386a5fbb70ba271886dcc1f3ca0f7

**Comment by gromgit** [comment]
>```
>==> Caveats
>Copy the example configuration into your home directory:
>cp /opt/homebrew/opt/spacebar/share/spacebar/examples/spacebarrc ~/.spacebarrc
>Logs will be found in
>/opt/homebrew/var/log/spacebar/spacebar.[out|err].log
>```

Have you looked for errors in the log files listed above?

**Comment by fightyz** [comment]
> > ```
> > ==> Caveats
> > Copy the example configuration into your home directory:
> > cp /opt/homebrew/opt/spacebar/share/spacebar/examples/spacebarrc ~/.spacebarrc
> > Logs will be found in
> > /opt/homebrew/var/log/spacebar/spacebar.[out|err].log
> > ```
> 
> Have you looked for errors in the log files listed above?

It works now.

I remembered that when using Homebrew 3.5.8 earlier(see [issue 13678](https://github.com/Homebrew/brew/issues/13678)), there was an similar error starting the service. The prompt mentioned: "Try re-running the command as root for richer errors", so I started the service as root.

Just now, when I entered `/opt/homebrew/var/log/spacebar/`, I found that the user group of the log file is root, and the error log prints a message like "spacebar can not run as root". So after I deleted the entire directory and reinstalled **spacebar**, the service could start.
